### PR TITLE
HMF tools: add LINX

### PR DESCRIPTION
**Pull request by vladsaveliev**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/hmftools-linx/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+TGT="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+[ -d "$TGT" ] || mkdir -p "$TGT"
+[ -d "${PREFIX}/bin" ] || mkdir -p "${PREFIX}/bin"
+
+cd "${SRC_DIR}"
+mv sv-linx*.jar $TGT/sv-linx.jar
+
+cp $RECIPE_DIR/LINX.sh $TGT/LINX
+ln -s $TGT/LINX $PREFIX/bin
+chmod 0755 "${PREFIX}/bin/LINX"
```
Comment (bgruening) [inline recipes/hmftools-linx/build.sh]: can we call it `linx` is capital writing needed?

File: recipes/hmftools-linx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - xorg-libxtst
+    - r-base ==3.6.*
+    - r-tidyr
+    - r-dplyr
+    - r-ggplot2
+    - r-cowplot
+    - r-magick
+    - bioconductor-gviz
+    - circos>=0.69.6                   # for circos
+    - libgd                            # for circos
```
Comment (bgruening) [inline recipes/hmftools-linx/meta.yaml]: cirocos should bring this in on there own, you don't need to specify it here I think

File: recipes/hmftools-linx/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  url: https://github.com/hartwigmedical/hmftools/releases/download/sv-linx-v{{ version }}/sv-linx_{{ version }}.jar
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - zlib
```
Comment (bgruening) [inline recipes/hmftools-linx/meta.yaml]: Can we remove zlib here? Why is this needed?

File: recipes/hmftools-linx/build.sh (hunk 1 above)
Comment (vladsaveliev) [reply recipes/hmftools-linx/build.sh]: Not really, just following the style of other Hartwig tools ([purple](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/hmftools-purple/meta.yaml), amber, cobalt). @chapmanb, wondering if there was a specific reason to capitalize? Happy to help changing it for all tools if it makes sense.

**Comment by vladsaveliev** [comment]
Thanks so much @bgruening for the review! Will address asap.

**Comment by dpryan79** [comment]
Please remove the jar file, it will be downloaded when the package is built and doesn't need to be in the recipe itself.

**Comment by vladsaveliev** [comment]
Sorry, added the jar accidentally; downloaded it into cwd when checking the sha256 sum and forgot to clean up. Should be good now.
